### ISSUE TITLE
Loading images from multiple fields of view

### ISSUE THREAD

**Issue by simone-codeluppi** [rbnvrw/nd2reader#3]
Hi
My output files have ~ 200 fields of view and the data are zxy with no time points. Is it possible to use pims to load directly the images from each field of view?

When I load the file:

```
with ND2Reader(fname) as images:
    print(images.sizes)

{'x': 2048, 'y': 2048, 't': 1, 'z': 45}
```
I get the correct information for one of the field of view. Even after I set ` images.bundle_axes = 'zyx'` I am still not able to access all the images in the file. 
I have the same issue if i load the file with the legacy.Nd2 module that I was using in the previous version of nd2reader.

I looked in the docs and also in pims and pims_nd2 but I couldn't find the relevant infos.

I will really appreciate any help.
Thanks a lot!

**Comment by rbnvrw** [comment]
Thank you for reporting the issue. I think this is not implemented yet if it is also not working with the legacy code - I will look into it if you can send me an example file via WeTransfer.

**Comment by rbnvrw** [comment]
I have fixed this in release 3.0.6.
